lizmap/modules/lizmap/lib/Project/QgisProject.php

```diff
@@ -868,7 +863,10 @@ public function getPrintTemplates()
         return $printTemplates;
     }
 
-    public function readLocateByLayers(&$locateByLayer)
```

Comment:
@laurentj `&` is no more necessary?

Reply:
no, because `$locateByLayer` is supposed to be an object, and an object is always pass by reference